tmp/bento-20220222-112500.log

```diff
@@ -0,0 +1,22 @@
+<14>1 2022-02-22T17:25:00.391Z - - 2150 - - DEBUG: (Model Converter) Schema file input: ../../ICDC/icdc-model-tool/model-desc/icdc-model.yml
```

Comment:
Is this log file required?

Reply:
no. Auto generated by model converter tool